docs/content/concepts/transfers/transfer-policies.mdx

```diff
@@ -0,0 +1,280 @@
+---
+title: Transfer Policies
+description: Learn about transfer policies on Sui.
+keywords: [ transfer, policy, transferpolicy, transfer receipt, transfer request, transferrequest ]
+---
+
+A [`TransferPolicy`](https://github.com/MystenLabs/sui/blob/65895261feba7a1241cc5342d44cf72386c5ebe0/crates/sui-framework/docs/sui/transfer_policy.md) on Sui is a customizable primitive that enables the owner of a type to create custom rules that define how the type can be transferred. A `TransferPolicy` can be used within a Sui Kiosk marketplace or any other system that integrates the `TransferPolicy` primitive. The creator can set any number of rules, such as paying a royalty or commission, and all must be satisfied in a single transaction for the transfer to succeed.
+
+In a kiosk, creating and sharing a `TransferPolicy<T>` makes the type `T` tradable in that kiosk. On every kiosk purchase, the `TransferPolicy` must confirm the `TransferRequest`, or the transaction fails.
+
+## Transfer policies for kiosks 
+
+When a kiosk purchase occurs, the system creates a `TransferRequest` "hot potato," and only the matching `TransferPolicy` can confirm it to unblock the transaction.
+
+A kiosk can trade an item of type `T` only if the `TransferPolicy` for `T` exists and the buyer can access it. A purchase issues a `TransferRequest` that must be resolved in a matching `TransferPolicy`. If no policy exists or the buyer can't access it, the transaction fails.
+
+This system gives creators maximum freedom and flexibility. By removing transfer policy logic from the trading primitive, only creators can set policies, and they control all enforcement as long as the primitive is used.
+
+## `TransferPolicy` rules 
+
+By default, a single `TransferPolicy` enforces nothing and requires no user action. It confirms `TransferRequests` and therefore unblocks a transaction. However, the system allows setting *rules*. A rule is a way to request an additional action from the user before the request can be confirmed.
+
+The rule logic is simple: someone can publish a new rule module, such as a "fixed fee," and add it to the `TransferPolicy`. Once the rule is added, `TransferRequest` needs to collect a `TransferReceipt` marking that the requirement specified in the rule was completed.
+
+### Example rule 
+
+For example, to implement VAT fees on every merchant transaction, you must introduce a rule.
+
+A rule needs to provide 4 main components:
+
+1. A `RuleWitness` struct which uniquely identifies the rule.
+1. A `config` type which is stored in the `TransferPolicy` and is used to configure the rule.
+1. A set function which adds the rule to the `TransferPolicy`. This must be performed by the `TransferPolicyCap` holder.
+1. An actionable function which adds the receipt into the `TransferRequest` and potentially adds to the `TransferPolicy` balance if the functionality involves some monetary transaction.
+
+```
+module examples::dummy_rule {
```

Comment:
Would be nice if these code examples existed in the repo so we could import them. The syntax changes faster than a screenshot.